### PR TITLE
Admin Panel - All Holidays

### PR DESCRIPTION
**Pull request by William-Unosquare**
This just lists all the holidays with their current status. Not incredibly useful as this grows but this will be a good base to work off in the future for some sort of holiday filtering, and yet another thing for demoing.

### CODE REVIEW

File: web-app/src/components/common/Errorbox/styled.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
 import styled from 'styled-components';
 
 export const ErrorContainer = styled.div`
+<<<<<<< HEAD
```
Comment (BrownMark) [inline web-app/src/components/common/Errorbox/styled.js]: Merging leftover tags.

**Review by BrownMark — CHANGES_REQUESTED** [review]
Just leftover merge tags

**Comment by William-Unosquare** [comment]
@BrownMark fixed - always the same file conflicting on me 😕

**Review by BrownMark — APPROVED** [review]
Loos good

**Pull request merged**